tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs

```diff
@@ -65,19 +65,26 @@ protected override async Task HandleCoreAsync(HandlerContext<CheckRunEventPayloa
                         (DateTimeOffset.UtcNow - payload.CheckRun.StartedAt).Days
                         );
                 }
+                else if (payload.CheckRun.Conclusion == new StringEnum<CheckConclusion>(CheckConclusion.Neutral))
```

Comment:
Do we want to bail on all non-successful check runs?
`Can be one of success, failure, neutral, cancelled, skipped, timed_out, or action_required`
https://docs.github.com/en/free-pro-team@latest/rest/reference/checks#runs

Reply:
Good question. I think they are mostly examples of what we would consider a failure case (not sure about skipped). If we want to make Check Enforcer report failures we'd probably want to think about how we handle each of those cases.

Even though Azure Pipelines doesn't use those conclusions, it doesn't mean that another check type couldn't.